`cloudformation.ResponseSerializationSuite` test failure

Ping @bpholt.

https://github.com/typelevel/feral/runs/4835158804?check_suite_focus=true#step:10:41

```
==> X feral.lambda.cloudformation.ResponseSerializationSuite.CloudFormationCustomResource should PUT the response to the given URI 1.98s munit.FailException: Failing seed: TMdL2IA_H4DCXl0kbz5HogldPyArS7jjBPgancWELoF=
You can reproduce this failure by adding the following override to your suite:

  override def scalaCheckInitialSeed = "TMdL2IA_H4DCXl0kbz5HogldPyArS7jjBPgancWELoF="

Exception raised on property evaluation.
> ARG_0: feral.lambda.LambdaEnv$$anon$2@18b
> Exception: java.lang.AssertionError: assertion failed

expect(body eqv expectedJson)
       |    |   |
       |    |   {
  "Status" : "FAILED",
  "Reason" : "unexpected CloudFormation request type `⮖並둈턴鎫恦暔厓ၯ럺炼攮ર꬏\r줅贷龤∣쐬텉塄戾Ⴁ欷老⫯ᘋ䗔쭤屑ऩ㍘ℎ扢⼼뺛鼟㬻붮詃汼圳ὐ䚓住ꮟ썵芓첲ꃻ훘폓磹磻ᯉ覦與匲糧̪픀杵㓼鍵ᱪﰩ痢ṱ褕쁽ꤸᵻ袖ᇘ條᝵䈴牑ꛀ麢㜷츝鳸꦳귐ยጷ`",
  "PhysicalResourceId" : "<",
  "StackId" : "ဉ",
  "RequestId" : "fe87926c-e1b8-43f1-b7e3-61f83a34a0bc",
  "LogicalResourceId" : "",
  "Data" : {
    "StackTrace" : [
      "java.lang.IllegalArgumentException: unexpected CloudFormation request type `⮖並둈턴鎫恦暔厓ၯ럺炼攮ર꬏\r줅贷龤∣쐬텉塄戾Ⴁ欷老⫯ᘋ䗔쭤屑ऩ㍘ℎ扢⼼뺛鼟㬻붮詃汼圳ὐ䚓住ꮟ썵芓첲ꃻ훘폓磹磻ᯉ覦與匲糧̪픀杵㓼鍵ᱪﰩ痢ṱ褕쁽ꤸᵻ袖ᇘ條᝵䈴牑ꛀ麢㜷츝鳸꦳귐ยጷ`"
    ]
  }
}
       |    false
       {
  "Status" : "FAILED",
  "Reason" : "unexpected CloudFormation request type `⮖並둈턴鎫恦暔厓ၯ럺炼攮ર꬏\r줅贷龤∣쐬텉塄戾Ⴁ欷老⫯ᘋ䗔쭤屑ऩ㍘ℎ扢⼼뺛鼟㬻붮詃汼圳ὐ䚓住ꮟ썵芓첲ꃻ훘폓磹磻ᯉ覦與匲糧̪픀杵㓼鍵ᱪﰩ痢ṱ褕쁽ꤸᵻ袖ᇘ條᝵䈴牑ꛀ麢㜷츝鳸꦳귐ยጷ`",
  "PhysicalResourceId" : "<",
  "StackId" : "ဉ",
  "RequestId" : "fe87926c-e1b8-43f1-b7e3-61f83a34a0bc",
  "LogicalResourceId" : "",
  "Data" : {
    "StackTrace" : [
      "java.lang.IllegalArgumentException: unexpected CloudFormation request type `⮖並둈턴鎫恦暔厓ၯ럺炼攮ર꬏"
    ]
  }
}

[error] Failed: Total 1, Failed 1, Errors 0, Passed 0
[error] Failed tests:
[error] 	feral.lambda.cloudformation.ResponseSerializationSuite
[error] (lambdaCloudFormationCustomResourceJS / Test / test) sbt.TestsFailedException: Tests unsuccessful
[error] Total time: 10 s, completed Jan 17, 2022 12:55:47 AM
```

--- a/lambda-cloudformation-custom-resource/src/main/scala/feral/lambda/cloudformation/CloudFormationCustomResource.scala
+++ b/lambda-cloudformation-custom-resource/src/main/scala/feral/lambda/cloudformation/CloudFormationCustomResource.scala
@@ -17,8 +17,7 @@
 package feral.lambda
 package cloudformation
 
-import cats.ApplicativeThrow
-import cats.MonadThrow
+import cats._
 import cats.syntax.all._
 import feral.lambda.cloudformation.CloudFormationRequestType._
 import io.circe._
@@ -28,9 +27,6 @@ import org.http4s.Method.PUT
 import org.http4s.circe._
 import org.http4s.client.Client
 import org.http4s.client.dsl.Http4sClientDsl
-
-import java.io.PrintWriter
-import java.io.StringWriter
 
 trait CloudFormationCustomResource[F[_], Input, Output] {
   def createResource(input: Input): F[HandlerResponse[Output]]
@@ -93,12 +89,8 @@ object CloudFormationCustomResource {
       Data = res.data.asJson
     )
 
-  private def stackTraceLines(throwable: Throwable): List[String] = {
-    val writer = new StringWriter()
-    throwable.printStackTrace(new PrintWriter(writer))
-
-    // TODO figure out how to include more of the stack trace, up to a total response size of 4096 bytes
-    writer.toString.linesIterator.toList.take(1)
-  }
+  // TODO figure out how to include more of the stack trace, up to a total response size of 4096 bytes
+  private def stackTraceLines(throwable: Throwable): List[String] =
+    List(throwable.toString)
 
 }

